src/wallet.js

```diff
@@ -154,6 +154,30 @@ class ThorWallet {
         }
     }
 
+    buildCallTransaction = async (clauses, options) => {
```

Comment:
do we need to have specific method for that or we can enhance buildTransaction one ?